modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImpl.java

```diff
@@ -838,33 +838,34 @@ private CompletableFuture<Void> close(boolean cancel) {
                                 : closeLocalFragments();
 
                         var finalStepFut = cancelResult.whenComplete((r, e) -> {
-                            queryManagerMap.remove(ctx.queryId());
+                            if (e != null) {
+                                Throwable ex = unwrapCause(e);
 
-                            try {
-                                ctx.cancel().cancel();
-                            } catch (Exception ignored) {
-                                // NO-OP
+                                LOG.warn("Exception raised during close", ex);
```

Comment:
Log message should be rephrased.

According to our [guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Java+Logging+Guidelines#JavaLoggingGuidelines-WARN):
```
WARN is what the team will need to review at the start of the next business day, at a high priority. WARN is always actionable - the user must understand how to react
```

From the message I don't understand what I should to do. TBH, I don't even understand what was going wrong. We closed a door and exception had been raised? A window?